doc/OVA_to_OVF.rst

```diff
@@ -0,0 +1,24 @@
+===========================
+Conversion from OVA to OVF
+===========================
+
+Follow the below mentioned procedure to convert an OVA file to an OVF file.
+
+#. Download and install the VMware OVF tool.
+
+#. After the installation is complete, navigate to the directory where you installed the OVF tool using the CD command. By default, the location to the directory is as follows.
```

Comment:
Should say, "For example, on a Windows machine, the default location is ..."

Reply:
Incorporated.